tools/ci/retry.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env pwsh
+
+if ($args.Length -lt 1) {
+    Write-Host "Usage: $($MyInvocation.InvocationName) <command> [arguments]"
+    Write-Host "The script will execute the provided command and retry in case of failures"
+    Exit 1
+}
+
+$NB_RETRIES = 5
+$command = [string]::Join(' ', $args)
+
+for ($i = 1; $i -le $NB_RETRIES; $i++) {
+    Invoke-Expression $command
+    $errorCode = $LASTEXITCODE
+    if ($errorCode -eq 0) {
+        Exit 0
+    } else {
```

Comment:
is the else necessary in the PS script?
if not there is not need, because of the `Exit`

Reply:
Forgot to remove it when I was playing around with it 👍